plugins/redis/secret_key.go

```diff
@@ -0,0 +1,55 @@
+package redis
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func SecretKey() schema.CredentialType {
```

Comment:
If this is specific for Redis Cloud, can we add that to the name?

Something like:
```suggestion
func RedisCloudAPIKey() schema.CredentialType {
```